### ISSUE TITLE
Getting "no such app" inside ChatGPT

### ISSUE THREAD

**Issue by suntereo** [Sogody/google-chatgpt-plugin#10]
Really excited about your plugin!

I just got GPT Plugin access last night and wanted to give your project a go!

I installed to Heroku by clicking on the one-click button [here](https://github.com/Sogody/google-chatgpt-plugin#installation-heroku).

But after installing the plugin, I'm getting this error:

<img width="527" alt="Xnip2023-05-11_19-09-32" src="https://github.com/Sogody/google-chatgpt-plugin/assets/171720/069548d8-210b-43c3-a687-e8b6473ca9bf">

I also do not see a good response when I hit the /search endpoint directly:

<img width="670" alt="Xnip2023-05-11_19-11-20" src="https://github.com/Sogody/google-chatgpt-plugin/assets/171720/6a765085-e0d0-4dc4-8730-e41f816025a2">

I'm wondering if there's a bug. Not sure what else I can check. 
Please let me know if you need any more info. Thanks!

**Comment by suntereo** [comment]
I just checked again and now the plug-in error matches when I directly hit the endpoint:

![IMG_1393](https://github.com/Sogody/google-chatgpt-plugin/assets/171720/8dcbc621-e952-4cb4-9825-f7594336f91c)

**Comment by suntereo** [comment]
I figured out my problem. I needed to enable the custom search engine on Google’s end (duh!). With that, I finally managed to get it working:

![IMG_1394](https://github.com/Sogody/google-chatgpt-plugin/assets/171720/559eabbc-03c0-489e-a6a5-9b4542a46e9c)

However, it only successfully completes every few attempts. I still get app not found and missing manifest files errors more often than success. 

Wondering if there is some type of caching problem. But I think the plug-in itself is working so I’m closing.
